etc/submit.sh

```diff
@@ -160,6 +160,7 @@ $pythonCommand Unpacker.py --sandbox=$SANDBOX --package=JobPackage.pkl --index=$
 cd job
 export WMAGENTJOBDIR=$PWD
 export PYTHONPATH=$PYTHONPATH:$PWD/WMCore.zip:$PWD
+export PYTHON27PATH=$PYTHON27PATH:$PWD/WMCore.zip:$PWD  # CMSSW_10_1_0 no longer uses PYTHONPATH
```

Comment:
I would suggest just changing 162-163 into:

if [ -z "$PYTHONPATH" ]
then
  export PYTHONPATH=$PWD/WMCore.zip:$PWD
else
  export PYTHONPATH=$PYTHON27PATH:$PWD/WMCore.zip:$PWD
fi

or some improved version of this
[ps, for later - do you really need to add $PWD?]

Reply:
after more checking it seems my suggestion should not be needed.